### ISSUE TITLE
Failing since switch to chromedriver (Unable to locate element: {"method":"class name","selector":"edgesearch"})

### ISSUE THREAD

**Issue by jbasoo** [xMNG/Microsoft-Rewards-Bot#15]
Running macOS Mojave. For now I've disabled current_edge_points function calls to keep it working.

If I visit https://www.bing.com/rewardsapp/bepflyoutpage?style=chromeextension in browser the .edgesearch element doesn't exist. Perhaps the element only appears for some accounts? For mine on the url above, the pcsearch is max 102 points, combined of pc and edge searches. In the rewards breakdown popup on the dashboard it separates this data into pc (90) and edge (12).

```
2019-02-06 01:53:50,423 :: INFO :: root :: Search Start
2019-02-06 01:53:58,271 :: ERROR :: root :: WebDriverException while executing pc portion
Traceback (most recent call last):
  File "/diskpath/Microsoft-Rewards-Bot/ms_rewards.py", line 800, in <module>
    search(search_list)
  File "/diskpath/Microsoft-Rewards-Bot/ms_rewards.py", line 460, in search
    if get_point_total(pc=True):
  File "/diskpath/Microsoft-Rewards-Bot/ms_rewards.py", line 673, in get_point_total
    browser.find_element_by_class_name('edgesearch').text.split('/'))
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 564, in find_element_by_class_name
    return self.find_element(by=By.CLASS_NAME, value=name)
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"class name","selector":"edgesearch"}
  (Session info: headless chrome=71.0.3578.98)
  (Driver info: chromedriver=2.46.628411 (3324f4c8be9ff2f70a05a30ebc72ffb013e1a71e),platform=Mac OS X 10.14.2 x86_64)
```

**Comment by xMNG** [comment]
What is the class name for the edge points node?

**Comment by jbasoo** [comment]
On https://www.bing.com/rewardsapp/bepflyoutpage?style=chromeextension there is no edge points node at all. It combines both edge and pc points into the `.pcsearch` node.

On https://account.microsoft.com/rewards/pointsbreakdown it's the first node in `#userPointsBreakdown` popup. It doesn't have a unique class or id but does mention edge in the content of .title-detail

For me, disabling the edge points search makes everything work fine. Perhaps making the edge points search optional and not throw an exception would be the easiest fix.

**Comment by ShoGinn** [comment]
@jbasoo I agree with you, having an edge search isn't beneficial since most of the time pcsearchs with this script will always be edge..

**Comment by xMNG** [comment]
Agreed. At the start of this project, I encountered situations where pc search would complete but edge search would not. After making some changes, this problem no longer occurs and I think it would safe to remove, which will done in the next patch. 

This would improve functionality for users outside of North America (it looks like).

Thanks @jbasoo , @ShoGinn

**Comment by jbasoo** [comment]
Awesome thanks @xMNG. Good work!

**Comment by ShoGinn** [comment]
Hopefully this never gets the eye of the ms lawyers and you get a cease and disist letter like the other bots.

**Comment by xMNG** [comment]
You know what's interesting? Look at who starred this repo.

**Comment by xMNG** [comment]
Also, the downside to removing edgesearch is that we lose the logging functionality. It would be ok for UK users but not NA. Thoughts?

**Comment by xMNG** [comment]
closed with update 2019.04.01, see also #35
